<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer_Button_Panel</class>
 <widget class="QWidget" name="Viewer_Button_Panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="button_panel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="save_bmp">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>save picture to BMP file</string>
        </property>
        <property name="text">
         <string>Save to BMP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_txt">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>save picture to TXT file</string>
        </property>
        <property name="text">
         <string>Save to TXT</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="horizontal_mirror">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>horizontal_mirror</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../img.qrc">
          <normaloff>:/horizontal</normaloff>:/horizontal</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="vertical_mirror">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>vertical_mirror</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../img.qrc">
          <normaloff>:/vertical</normaloff>:/vertical</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rotate_plus">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>+90º</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../img.qrc">
          <normaloff>:/rotate_plus</normaloff>:/rotate_plus</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="angle">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="specialValueText">
         <string/>
        </property>
        <property name="suffix">
         <string> °</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rotate_minus">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>-90º</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../img.qrc">
          <normaloff>:/rotate_minus</normaloff>:/rotate_minus</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_transform">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>reset transformation</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../img.qrc">
          <normaloff>:/reset</normaloff>:/reset</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scaled_plus">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>scale +1.1</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../img.qrc">
          <normaloff>:/plus</normaloff>:/plus</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scaled_minus">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>scale -1.1</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../img.qrc">
          <normaloff>:/minus</normaloff>:/minus</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../img.qrc"/>
 </resources>
 <connections/>
</ui>
